scripts/run_celery_no_sending.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
run_celery_core_tasks as an alternative name maybe?

Reply:
yeah, that's less clunky!